tests/unit/s2n_tls13_state_machine_handshake_test.c

```diff
@@ -305,6 +307,86 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_connection_free(conn));
     }
 
+    /* Test: TLS 1.3 handshakes all follow CCS middlebox compatibility rules.
+     * RFC: https://tools.ietf.org/html/rfc8446#appendix-D.4 */
```

Comment:
Thanks for linking to what you're testing! It really helps to give context around the intent.